Provider/build.gradle.kts

```diff
@@ -85,6 +85,48 @@ publishing {
                     }
                 }
             }
+            pom {
+                name.set("Confidence SDK Provider Android")
+                description.set("Confidence SDK Provider Android")
+                url.set("https://confidence.spotify.com/")
```

Comment:
in the [java pom](https://github.com/spotify/confidence-openfeature-provider-java/blob/main/pom.xml) we use the github url here... not at all sure what is more correct :)